tasks/offlinePackagingTasks.ts

```diff
@@ -206,13 +206,14 @@ async function acquireNugetPackage(packageName: string, packageVersion: string,
 }
 
 async function doPackageOffline() {
-    // Set the package version using git versioning.
+    // Set the package.json version based on the value in version.json.
     const versionInfo = await nbgv.getVersion();
     console.log(versionInfo.npmPackageVersion);
     await nbgv.setPackageVersion();
 
     let prerelease = false;
     if (argv.prerelease) {
+        console.log('Packaging prerelease version.');
```

Comment:
Did you want an 'else' to log the other way as well just so the logs are clear either way?